src/common/components/dashboard/dashboard.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+const styles: any = require('./dashboardStyles.scss');
+import {DashboardItemComponent, IDashboardItem} from './components/dashboardItem';
+
+interface IProps {
+  title: string;
+  items: IDashboardItem[];
+}
+
+export class DashboardComponent extends React.Component<IProps, {}> {
+  public render() {
+    return(
+      <div className={styles.dashboard}>
+        <h3 className={styles.dashboardTitle}>{this.props.title}</h3>
+        <div className={styles.dashboardItems}>
+          {
+            this.props.items.map((item) =>
+              <DashboardItemComponent
+                style={this.getItemsPerRowStyle(this.props.items)}
+                key={item.name}
+                item={item}
+              />,
+            )
+          }
+        </div>
+      </div>
+    );
+  }
+
+  private getItemsPerRowStyle(items: IDashboardItem[]): React.CSSProperties {
+    return items.length >= 3 ?
+      {flexBasis: '33.33%'} :
+      {flexBasis: `${100 / items.length}%`};
+  }
```

Comment:
Why this "magic" numbers? Add a comment at the top of this method indicating why we have chosen 33.33 and 3